.gitlab-ci.yml

```diff
@@ -25,3 +23,20 @@ release:
     - echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
     - npm publish ./dist/splunk-otel-${CI_COMMIT_REF_NAME:1}.tgz
     - rm -f ~/.npmrc
+
+prerelease:
+  stage: release
+  artifacts:
+    paths:
+      - dist/
+  rules:
+    - if: $CI_COMMIT_BRANCH =~ /^prerelease-.+/ && $CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+-.+/
```

Comment:
This means only tags on a specific branch will have this flow ran?

Reply:
Changed the yaml, the trigger is only based on tag names now. So it will be possible to create feature branches and publish experimental versions from there